types.go

```diff
@@ -641,6 +659,26 @@ type Order struct {
 	Reason         string           `json:"reason"`
 }
 
+// Position - Represents a single position
+type Position struct {
+	TransactionID      string  `json:"-"`
+	OrderTransactionID string  `json:"ordertxid"`
+	Pair               string  `json:"pair"`
+	TradeTime          float64 `json:"time"`
+	PositionType       string  `json:"type"`
+	OrderType          string  `json:"ordertype"`
+	Cost               float64 `json:"cost,string"`
+	Fee                float64 `json:"fee,string"`
+	Volume             float64 `json:"vol,string"`
+	VolumeClosed       float64 `json:"vol_closed,string"`
+	Margin             float64 `json:"margin,string"`
+	Value              float64 `json:"value,string"`
+	Net                float64 `json:"net,string"`
```

Comment:
from some tests i did, it seems this breaks when the net value is returned from kraken as a string with a `+` sign i.e. `+24.768` . Would it make sense to try a custom unmarhsaling ? Something maybe similar to how OrderBookItem is handled ?

Reply:
I didn't realize they do this, I can check later for this or if you have a patch we can merge it in!

Reply:
Not sure its the best way ... but thats what i used to get around it.
But then when parsing the object you need to do something like this, when using the Net property
```
net := float64(Position.Net)
```